sfn-query-large-datasets-cdk/app.py

```diff
@@ -0,0 +1,307 @@
+#!/usr/bin/env python3
+import json
+import random
+import string
+import os
+from aws_cdk import (
+    App,
+    Aspects,
+    Environment,
+    RemovalPolicy,
+    Stack,
+    aws_s3 as s3,
+    aws_iam as iam,
+    aws_sns as sns,
+    aws_kms as kms,
+    aws_glue as glue,
+    aws_athena as athena,
+    aws_stepfunctions as sfn,
+    aws_logs as logs,
+    CfnOutput as outputs
+)
+from constructs import Construct
+from cdk_nag import AwsSolutionsChecks, NagSuppressions
+
+
+class SfnQueryLargeDatasetsCdkStack(Stack):
+    def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
+        super().__init__(scope, construct_id, **kwargs)
+
+        # Random string generation for suffixes
+        random_string = "".join(random.choices(string.ascii_lowercase, k=8))
```

Comment:
Can we not rely on the default name generated by CDK and pass it as a reference when needed?

Reply:
Ah I see, inherited from the TF project.